File: project 1 - math/python/BMI.py
weight = float(input("enter your weight (kg): "))  
height = float(input("enter your height (m): "))  
result = weight / height ** 2  

if result < 18.5:  
    print("underweight")  
elif 18.5 <= result < 25:  
    print("normal weight")  
elif 25 <= result < 30:  
    print("overweight")  
elif 30 <= result < 35:  
    print("obese")  
elif 35 <= result < 40:  
    print("extra obese")  
else:  
    print("severely obese")

File: project 1 - math/js/BMI.js
 
let weight = parseFloat(prompt("Enter your weight (kg):"));  
let height = parseFloat(prompt("Enter your height (m):"));  

 
let bmi = weight / (height ** 2);  

 
if (bmi < 18.5) {  
    console.log("Underweight");  
} else if (bmi >= 18.5 && bmi < 25) {  
    console.log("Normal weight");  
} else if (bmi >= 25 && bmi < 30) {  
    console.log("Overweight");  
} else if (bmi >= 30 && bmi < 35) {  
    console.log("Obese");  
} else if (bmi >= 35 && bmi < 40) {  
    console.log("Extra obese");  
} else {  
    console.log("Severely obese");  
}
